it/google-cloud-platform/src/test/java/org/apache/beam/it/gcp/bigquery/BigQueryIOST.java

```diff
@@ -84,6 +84,7 @@ public final class BigQueryIOST extends IOStressTestBase {
   private static final String READ_ELEMENT_METRIC_NAME = "read_count";
   private static final String STORAGE_WRITE_API_METHOD = "STORAGE_WRITE_API";
   private static final String STORAGE_API_AT_LEAST_ONCE_METHOD = "STORAGE_API_AT_LEAST_ONCE";
+  private static final int STORAGE_API_AT_LEAST_ONCE_MAX_ALLOWED_DIFFERENCE = 10_000;
```

Comment:
I imagine the difference could be linear to the total number of elements. How about allowed different fraction?